openadapt/privacy/base.py

```diff
@@ -5,7 +5,7 @@
 from PIL import Image
 from pydantic import BaseModel  # pylint: disable=no-name-in-module
 
-from openadapt.config import config
+from openadapt.config import SCRUB_KEYS_HTML, config
```

Comment:
Can you please clarify why it's preferable to move this outside of the config object?